
From: Andi Kleen <ak@suse.de>

Enlarge i386/x86-64 kernel command line to 2k

This is useful when the kernel command line is used to pass other
information to initrds or installers.

On i386 it was duplicated for unknown reasons.

Signed-off-by: Andi Kleen <ak@suse.de>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/asm-i386/param.h   |    2 +-
 25-akpm/include/asm-i386/setup.h   |    2 +-
 25-akpm/include/asm-x86_64/setup.h |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff -puN include/asm-i386/param.h~x86_64-i386-increase-command-line-size include/asm-i386/param.h
--- 25/include/asm-i386/param.h~x86_64-i386-increase-command-line-size	Wed Jan 12 15:46:10 2005
+++ 25-akpm/include/asm-i386/param.h	Wed Jan 12 15:46:10 2005
@@ -18,6 +18,6 @@
 #endif
 
 #define MAXHOSTNAMELEN	64	/* max length of hostname */
-#define COMMAND_LINE_SIZE 256
+#define COMMAND_LINE_SIZE 2048
 
 #endif
diff -puN include/asm-i386/setup.h~x86_64-i386-increase-command-line-size include/asm-i386/setup.h
--- 25/include/asm-i386/setup.h~x86_64-i386-increase-command-line-size	Wed Jan 12 15:46:10 2005
+++ 25-akpm/include/asm-i386/setup.h	Wed Jan 12 15:46:10 2005
@@ -17,7 +17,7 @@
 #define MAX_NONPAE_PFN	(1 << 20)
 
 #define PARAM_SIZE 2048
-#define COMMAND_LINE_SIZE 256
+#define COMMAND_LINE_SIZE 2048
 
 #define OLD_CL_MAGIC_ADDR	0x90020
 #define OLD_CL_MAGIC		0xA33F
diff -puN include/asm-x86_64/setup.h~x86_64-i386-increase-command-line-size include/asm-x86_64/setup.h
--- 25/include/asm-x86_64/setup.h~x86_64-i386-increase-command-line-size	Wed Jan 12 15:46:10 2005
+++ 25-akpm/include/asm-x86_64/setup.h	Wed Jan 12 15:46:10 2005
@@ -1,6 +1,6 @@
 #ifndef _x8664_SETUP_H
 #define _x8664_SETUP_H
 
-#define COMMAND_LINE_SIZE	256
+#define COMMAND_LINE_SIZE	2048
 
 #endif
_
